src/seedu/addressbook/AddressBook.java

```diff
@@ -255,34 +285,40 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
-            showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
-            exitProgram();
-        }
-
-        if (args.length == 1) {
-            setupGivenFileForStorage(args[0]);
-        }
-
-        if(args.length == 0) {
-            setupDefaultFileForStorage();
-        }
+    	
+    	final int HAS_ONE_ARG = 1;
+    	final int HAS_NO_ARG = 0;
+    	
+    	switch (args.length){
+    		case (HAS_ONE_ARG) :
+    			setupGivenFileForStorage(args[0]);
+    			break;
+    			
+    		case (HAS_NO_ARG) :
+    			setupDefaultFileForStorage();
+    			break;
+    		
+    		default:
+    			showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
+    			exitProgram();
+    	}
```

Comment:
Misread the bot's error message, was initially having the impression that the code itself has an original format to follow or stick to; searched and retrieved one of the address book codes, which happened to be @xdrawks one, to edit the problem.